create-publication-from-scopus/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java

```diff
@@ -408,6 +409,30 @@ void shouldThrowExceptionWhenSrcTypeIsNotSuppoerted() throws IOException {
         assertThat(appender.getMessages(), containsString(expectedMessage));
     }
 
+    private static List<Serializable> contentWithSupInftagsScopus14244261628() {
+        //this is an actual title from doi: 10.1016/j.nuclphysbps.2005.01.029
+        return List.of("Non-factorizable contributions to ̄B",
+                       generateInf("d"),
+                       generateSup("0"),
+                       " → D",
+                       generateInf("s"),
+                       "(*) ̄D",
+                       generateInf("s"),
+                       "(*)");
+    }
+
```

Comment:
I landed on using this hardcoded example instead of the fuzzer, as the expected title would have to be generated in the same way as the implementation. This is also one of the ugliest examples I could find.